### ISSUE TITLE
The UI's js breaks when using Kaspersky on Windows

### ISSUE THREAD

**Issue by gplanchat** [akeneo/pim-community-dev#5232]
When using Kaspersky anti-virus, some JS are blocked and/or are executed with parse errors.

I'm pretty sure it is a false-positive from Kaspersky (the customer reported he had the same sort of bug on paypal.com's payment form), but as long as the anti-virus is deployed on the whole company, changing (or disabling) the anti-virus could be troublesome.

I'll post some more info and js backtraces later

Versions impacted : CE and EE (demos), at least 1.5 and 1.6

**Comment by fabienlem** [comment]
Hello @gplanchat,

Thank you for reporting this. It seems you are the first to report such anti-virus linked issue. Don't hesitate to provide us further information.

Have a nice day!
Fabien

**Comment by a2xchip** [comment]
@gplanchat @fabienlem I will send question about anti-virus software used to users reported such js blocking, if confirmed will mean at least from 1.4

**Comment by gplanchat** [comment]
The issue is present on the 1.5 EE demo.

Here is the console trace :

```
SyntaxError: Unexpected token t in JSON at position 223
    at JSON.parse (<anonymous>)
    at Function.parseJSON (http://demo-ee.akeneo.com/bundles/pimui/lib/jquery/jquery-1.10.2.js:550:23)
    at child.getCorrectedData (http://demo-ee.akeneo.com/bundles/pimnavigation/js/navigation.js:354:29)
    at child.handleResponse (http://demo-ee.akeneo.com/bundles/pimnavigation/js/navigation.js:384:127)
    at child.<anonymous> (http://demo-ee.akeneo.com/bundles/pimnavigation/js/navigation.js:169:30)
    at fire (http://demo-ee.akeneo.com/bundles/pimui/lib/jquery/jquery-1.10.2.js:3048:30)
    at Object.fireWith [as resolveWith] (http://demo-ee.akeneo.com/bundles/pimui/lib/jquery/jquery-1.10.2.js:3160:7)
    at done (http://demo-ee.akeneo.com/bundles/pimui/lib/jquery/jquery-1.10.2.js:8235:14)
    at XMLHttpRequest.callback (http://demo-ee.akeneo.com/bundles/pimui/lib/jquery/jquery-1.10.2.js:8778:8)
```

```
handleResponse @ navigation.js:421
(anonymous function) @ navigation.js:169
fire @ jquery-1.10.2.js:3048
fireWith @ jquery-1.10.2.js:3160
done @ jquery-1.10.2.js:8235
callback @ jquery-1.10.2.js:8778
```

**Comment by mickaelandrieu** [comment]
Hi @gplanchat, thanks for report.

Did you manage to fix it for your customer(s)? Can you provide a patch?

Regards,

**Comment by Doodoune** [comment]
Hello!

I close this issue as it was solved since 1.7, feel free to reopen it if you still have an error.

Regards,
Elodie
